lib/brexit_checker/questions.yaml

```diff
@@ -205,15 +205,10 @@ questions:
     type: "single_wrapped"
```

Comment:
Should we change the question type here?

Reply:
Ah interesting, what does question type do?
Because this has switched to radio buttons.

Reply:
It's probably a minor thing, as I can't think of any objective issue with leaving as-is. Perhaps we should condense the question types here: https://github.com/alphagov/finder-frontend/blob/master/app/views/brexit_checker/show.html.erb#L58

Reply:
Should I bung this in the backlog? or a ticket on planning?

Reply:
I'd say only if it's causing actual pain in development. Not sure if that's the case?